thunder/torch/__init__.py

```diff
@@ -5754,6 +5754,7 @@ def register_default_torch_op(torchfn: Callable, torch_module):
         name=torchfn_name,
         meta=_fn,
         id=f"{torch_module.__name__}.{torchfn_name}",
+        is_prim=True,
```

Comment:
Interesting! Why do we want to change this? Won't this default torch operations to be primitives? What's the effect of that?

Reply:
Note that these are the auto-registered operations that don't have a decomposition into subsymbols. Arguably, this has been a bug before, as dce on a symbol with auto-registered operations as subsymbols would remove them.